PCbuild/_freeze_module.vcxproj

```diff
@@ -446,6 +447,14 @@
                       Overwrite="true"
                       Condition="$(PyConfigHText.TrimEnd()) != $(OldPyConfigH.TrimEnd())" />
   </Target>
+  <Target Name="_BuildJIT" BeforeTargets="PrepareForBuild" Condition="'$(UseJIT)' == 'true'" DependsOnTargets="_UpdatePyconfig;FindPythonForBuild">
+    <PropertyGroup>
+      <JITArgs Condition="$(Platform) == 'Win32'">i686-pc-windows-msvc</JITArgs>
+      <JITArgs Condition="$(Platform) == 'x64'">x86_64-pc-windows-msvc</JITArgs>
+      <JITArgs Condition="$(Configuration) == 'Debug'">$(JITArgs) --debug</JITArgs>
+    </PropertyGroup>
+    <Exec Command='$(PythonForBuild) "$(PySourcePath)Tools\jit\build.py" $(JITArgs)' WorkingDirectory="$(IntDir)"/>
+  </Target>
```

Comment:
Sounds good. I was just following the example set by the free-threading PRs to orient myself. Probably the `Py_GIL_DISABLED` stuff just above isn't needed either, then?

Reply:
`Py_GIL_DISABLED` is a bit more special because it modifies `pyconfig.h`, so yeah, that one should stay. `_Py_JIT` shouldn't need to be present in all extension modules, which is simpler to handle.